### ISSUE TITLE
Insurance and Address Services are broken.

### ISSUE THREAD

**Issue by sjpadgett** [openemr/openemr#5357]
### Describe the bug
On new insurance company insert, false is always return causing the new fresh id to not be returned and a new address entry is missed.

The workflow assumption is that after insert, the new table record id is fetched from the newly created record. If only this was true and normally it would be. 

However, insurance_companies and address table do not use auto increment thus will not generate a new record id whereas the returned value from action will be 0 or false. Current logic traps for empty as a failed insert causing a returned false.

Simply put, if table id is not an auto increment then doesn't have a value to return. We should get a helpful die on fail however, unsure that will percolate to user.

**Comment by sjpadgett** [comment]
@stephenwaite Now that you moved the payer type you'll need a way to input in Insurance company. Unless c(1) doesn't ask for payer.

**Comment by stephenwaite** [comment]
added it with CQM source of payment
![Screenshot from 2022-05-18 18-43-03](https://user-images.githubusercontent.com/5618372/169167393-75d7717b-3b87-4ff6-9e67-39af941767d4.png)

**Comment by sjpadgett** [comment]
Groovy. I must have missed the source as I don't have. Will look for it.

**Comment by sjpadgett** [comment]
nice @stephenwaite thanks.
